src/cryptsetup/cryptsetup-tokens/meson.build

```diff
@@ -1,7 +1,8 @@
 # SPDX-License-Identifier: LGPL-2.1-or-later
 
-cryptsetup_token_sym = files('cryptsetup-token.sym')
-cryptsetup_token_sym_path = meson.current_source_dir() / 'cryptsetup-token.sym'
+if conf.get('HAVE_LIBCRYPTSETUP_PLUGINS') != 1
+        subdir_done()
+endif
```

Comment:
Why not just put `conditions : ['HAVE_LIBCRYPTSETUP_PLUGINS'` in the definition?